docs/integrations/destinations/kinesis.md

```diff
@@ -35,8 +35,8 @@ The connector buffer size should also be tweaked according to your data size and
 
 * The connector is compatible with the latest Kinesis service version at the time of this writing. 
 * Configuration
-    * **_Endpoint_**(`Optional`): Aws Kinesis endpoint to connect to. Default endpoint if not provided   
```

Comment:
what is the reason for removing `(Optional)`?

Reply:
The task is to set "`endpoint`" and "`region`" as required fields, so I changed `spec` and removed `Optional` from the doc